scripts/src/test.sh

```diff
@@ -120,7 +120,7 @@ doc_test() {
 }
 
 time_consuming_tests() {
-  $CARGO test -p demo-fungible-token --no-fail-fast "$@" -- --nocapture --ignored
+  $CARGO test -p demo-fungible-token --no-fail-fast --release -- --nocapture --ignored
```

Comment:
@ark0f any ideas?

Reply:
this is because `demo-fungible-token` runs stress tests and performance degrades with debug profile
the second line (wasm-builder tests) fails in release mode